Automattic/wp-calypso
# Framework: Remove unused VIP sites

While working on #11352 I discovered that there is `vip` section prototype that has never been exposed to non-development envs. I double checked the current status with @scottsweb and this is what he answered:

> We are not currently using those files and any further development there is at least 3 months out. If you would like to remove them then please do. I imagine a great deal has changed in Calypso since they were put together :) 

This PR removes all feature flags and code that was meant to power `vip` section.

### Testing
1. Run `make test`. Make sure all tests pass.
2. Run `make run`. Make sure Calypso still loads.
3. Go to `My Sites` section and make sure sidebar works as before.

### Review
